libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -112,9 +112,6 @@ AP_AHRS_DCM::update(bool skip_ins_update)
     // update trig values including _cos_roll, cos_pitch
     update_trig();
 
-    // update AOA and SSA
-    update_AOA_SSA();
-
```

Comment:
i'd prefer to drop the update in the accessors

Reply:
Done